.github/workflows/start-binary.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
       - name: Attempt to start binary
         run: |
-          ./build/defradb start &
+          ./build/defradb start --no-keyring &
```

Comment:
question: Why do we need to start with this flag?

Reply:
todo: Might be worth adding instructions for this extra configuration step in the root `README.md` (perhaps in the configuration section?) where as that is the default, and mention this flag to skip the keyring configuration.